Services/Interfaces/web_interface/controllers/community_authentication.py

```diff
@@ -45,7 +47,12 @@ def community_login():
                     log_exceptions=False
                 )
                 logged_in_email = form.email.data
-                return flask.redirect('community')
+                if after_login_action == "sync_account":
+                    added_profiles = models.sync_community_account()
+                    if added_profiles:
+                        flask.flash(f"Downloaded {len(added_profiles)} profile{'s' if len(added_profiles) > 1 else ''} "
```

Comment:
It is displayed to the user?

Reply:
yes, as here https://user-images.githubusercontent.com/9078616/209992555-ec9ebcad-1550-4894-a38f-596c275947b7.png